connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -275,6 +288,9 @@ export async function crawlWebsiteByConnectorId(connectorId: ModelId) {
         });
 
         try {
+          if (extracted.length > MAX_SMALL_DOCUMENT_TXT_LEN) {
```

Comment:
nit: why small?
imo "max_document_length" would be the cleanest

Reply:
this parameter is data_sources-wide and not specific to the crawler, it comes from #6411